.github/workflows/galaxy_ci.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: Galaxy CI
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'roles/galaxy_post_install/**'
+      - 'roles/pulp_devel/tasks/galaxy_ui.yml'
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '15 2 * * *'
+jobs:
+  core:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      # Testing different python versions because of
+      # https://pulp.plan.io/issues/5768#note-17
+      matrix:
+        include:
+          - ansible_core: ansible-core
+          - ansible_core: git+https://github.com/ansible/ansible.git
+    steps:
+      - uses: actions/checkout@v2.3.1
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+      - name: Install Ansible
+        run: |
+          pip install --upgrade pip
+          sudo apt remove ansible
+          pip install ${{ matrix.ansible_core }}
+      - name: Install Molecule
+        run: pip install "molecule!=3.3.1" docker molecule-docker ansible-lint[yamllint]
+      - name: Setting pulp.pulp_installer collection
+        run: |
+          make vendor
+          make install
+          ansible-galaxy collection install -p build/collections --force community.docker
+      - name: Molecule Test
+        run: |
+          ansible --version
+          cp .ci/assets/galaxy/group_vars/all molecule/source-static/group_vars/all
```

Comment:
After thinking about it a little bit (but not thoroughly), it would make more sense to create a new ansible molecule scenario, with a name like `source-static-galaxy`

So a new folder like: `molecule/source-static-galaxy`

This would enable us to run it locally on our laptops with a single `molecule` command, and for us to see that there is another scenario we can run locally.

Then document in the development section of the docs docs (the list of molecule scenarios.)

Reply:
I only thought about CI, because I don't like to review PRs that aren't tested, I never thought about running it locally

Reply:
I see what you mean.

In that case it is up to them.